cfme/containers/container.py

```diff
@@ -58,3 +58,7 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+    @staticmethod
+    def get_names():
```

Comment:
Also, keep in mind that this will only give you rows of the table on the first page. `CheckboxTable` doesn't paginate automatically through all pages so you will have to raise the number of displayed rows per page or just end up with only 20 (or whatever is the default number) names out of potentially hundreds / thousands.